### DISCUSSION TITLE
Option to not show entire output on screen in assertions

### DISCUSSION THREAD

**Discussion by ShrutiUpreti95** [artilleryio/artillery#1464] [Artillery]
Currently if I add assertions in my config, and run the tests, I can still see the output of my rest APIs on the screen. Is there an option to not show the entire output, and only the summary of tests run on the screen with the command line?

**Comment by hassy** [comment]
What are the options you're configuring the plugin with? (I'm assuming you're using `artilleryp-plugin-expect`?)

**Reply by ShrutiUpreti95** [reply]
Yes, I am using the expect plugin, and capturing the response from my REST APIs and validating it. Is there an option where I can avoid showing the expect values to be logged into the console?

**Reply by hassy** [reply]
This might be what you're looking for: https://www.artillery.io/docs/guides/plugins/plugin-expectations-assertions#formatters

Set `outputFormat` to `silent` to disable output from the plugin, i.e.:

```
config:
  plugins:
    expect:
      outputFormat: silent
```

**Reply by ShrutiUpreti95** [reply]
Thank you!
